tests/emukit/test_acquisitions.py

```diff
@@ -6,7 +6,8 @@
 from scipy.optimize import check_grad
 
 from bayesian_optimization.test_entropy_search import entropy_search_acquisition
-from emukit.bayesian_optimization.acquisitions import ExpectedImprovement, NegativeLowerConfidenceBound, EntropySearch, MaxValueEntropySearch
+from emukit.bayesian_optimization.acquisitions import ExpectedImprovement, NegativeLowerConfidenceBound, EntropySearch
+from emukit.bayesian_optimization.acquisition import MaxValueEntropySearch
```

Comment:
That's weird, why is it coming from a different package name here?

Reply:
No idea!
I had to split the line, as it was too long.

Reply:
Yeah, that's fine, it's just that first line has "acquisition_s_" and second "acquisition"